neqo-http3/src/server_events.rs

```diff
@@ -243,12 +243,23 @@ pub struct WebTransportRequest {
     stream_handler: StreamHandler,
 }
 
+#[derive(Debug, Clone)]
+pub struct ConnectUdpRequest {
+    stream_handler: StreamHandler,
+}
+
 impl Display for WebTransportRequest {
     fn fmt(&self, f: &mut Formatter) -> fmt::Result {
         write!(f, "WebTransport session {}", self.stream_handler)
     }
 }
 
+impl Display for ConnectUdpRequest {
```

Comment:
Ordering is a bit odd here.  Either all types first, then all implementations; or, all definitions for the same type together.